src/test/java/POM/pages/CurrentQueuePage.java

```diff
@@ -0,0 +1,31 @@
+package POM.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CurrentQueuePage extends BasePage {
+    //Web Elements
+    @FindBy(xpath="//section[@id='queueWrapper']//tr//td[@class='title']")
```

Comment:
Locator not found

![image](https://user-images.githubusercontent.com/104761482/200766154-33e16309-1a79-4b17-9c7a-bcac0ffdae84.png)

Try this: `//section[@id='queueWrapper']//..//td[@class='title']`

![image](https://user-images.githubusercontent.com/104761482/200766454-bf57f8a6-bd8a-4671-b7e7-26164aec6034.png)